1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,33 @@
+import { fizzbuzz } from "./fizzbuzz";
 
 describe("fizzbuzz", () => {
 
+    it("should return a string", () => {
+        expect(typeof fizzbuzz(1)).toBe("string");
+    });
+
+    it.each([3,9,42])("should return Fizz when given %i", (n) => {
+        expect(fizzbuzz(n)).toBe("Fizz");
+    });
+
+    it("should return Buzz when given 5", () => {
+        expect(fizzbuzz(5)).toBe("Buzz");
+    });
+
+    it("should return FizzBuzz when given 15", () => {
+        expect(fizzbuzz(15)).toBe("FizzBuzz");
+    });
+
+    it("should return 43 when given 43", ()=>{
+        expect(fizzbuzz(43)).toBe("43");
+    })
+
+    it("should return FizzBuzz when given 45", ()=>{
```

Comment:
You could combine this with the test for 15 above in `it.each()` because they're both tests for 'FizzBuzz'

Reply:
That is true but since it is just repetitive for the second time, I decided not to do that. But then I saw another implementation that utilizes an object of input, expected so that you can refactor every value test using that structure. That could be improved like that but I get the sentiment. 

Thanks for your time!